### ISSUE TITLE
[BITV] 9.1.1.1c/2.1 - Tile headers such as "Recommended files" contain decorative images which do not have alternative texts and are not marked as decorative elements with aria-hidden="true". (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36908]
- [x] Fix
- [x] Backport
- [x] to be added here as todo fort all default-shipped widgets

i.e. ```<div data-v-7d1a0f30="" role="img" class="icon-activity"></div>```

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3a2875951a-v2-n1

</details>

**Comment by JuliaKirschenheuter** [comment]
All images must have an ALT tag (aka ALT attribute). However, for decorative images, the ALT tag should be empty (a single empty space works, too). Screen readers and other technologies will recognize an empty ALT tag, identify the image as decorative and skip over it.
https://accessibility.asu.edu/articles/images#:~:text=Decorative%20images,-An%20image%20that's&text=All%20images%20must%20have%20an%20ALT%20tag%20(aka%20ALT%20attribute,decorative%20and%20skip%20over%20it.

https://www.portent.com/blog/design-dev/decorative-images-how-to-identify-and-tag-them-for-accessibility.htm#:~:text=If%20an%20image%20is%20just,aloud%20by%20a%20screen%20reader.

But a right implementation way would be: https://dequeuniversity.com/rules/axe/4.0/role-img-alt
